src/test/java/com/basicbug/bikini/feed/repository/FeedRepositoryTest.java

```diff
@@ -46,4 +56,41 @@ public void cleanup() {
         assertThat(feed.getFeedId()).isEqualTo(uuid);
         assertThat(feed.getContent()).isEqualTo(content);
     }
+
+    @Test
+    public void countByUser() {
+        // given
+        User user = getTestUser();
+        userRepository.save(user);
+
+        Feed feed1 = getTestFeed(user);
+        Feed feed2 = getTestFeed(user);
+        feedRepository.save(feed1);
+        feedRepository.save(feed2);
+
+        // when
+        long count = feedRepository.countByUser(user);
+
+        // then
+        assertThat(count).isGreaterThan(0);
```

Comment:
```suggestion
        assertThat(count).isEqualTo(2);
```

이렇게 좀 더 명확하게 테스트 해도 좋을 것 같아요